lib/ae_mdw/sync/async_tasks/store.ex

```diff
@@ -80,4 +81,16 @@ defmodule AeMdw.Sync.AsyncTasks.Store do
       {[], _cont} -> false
     end
   end
+
+  defp dedup_records(tasks) do
```

Comment:
As shown by `Store.save_new/2` test case, the code already dedups for new tasks (but some database might have duplicated already saved like mainnet.zip snapshot from Nov 10th).